### PR TITLE
Show managers

### PR DESCRIPTION
**Pull request by navisatebi**

### CODE REVIEW

File: src/integration-test/java/uk/ac/ebi/tsi/portal/api/team/TeamRestControllerIT.java [hunk 1]
```diff
@@ -136,34 +158,102 @@ public void getAllTeamsForCurrentUser() throws Exception {
...
+						.accept(MediaType.APPLICATION_JSON))
+				.andExpect(status().isOk())
+				.andExpect(jsonPath("$.managerEmails", hasSize(1)));
+	}
 
-		@Test
-		public void canRemoveMemberFromTeam() throws Exception{
+	@Test
+	@WithMockUser(username = "usr-b070585b-a340-4a98-aff1-f3de48da8c38")
+	public void non_team_owner_can_see_team_manager_emails() throws Exception {
```
Comment (aniewielska) [inline src/integration-test/java/uk/ac/ebi/tsi/portal/api/team/TeamRestControllerIT.java:180]: I am not sure, what is going on in the test.
1) Everybody should be able to retrieve some basic information about the team.
2) The name of test is misleading (are you checking that 'non_team_owner_can_see_team_manager_emails' - is that allowed? It should not be)
3) What is that assert in catch phrase actually doing? What kind of error are you expecting.

File: src/main/java/uk/ac/ebi/tsc/portal/api/team/service/TeamService.java [hunk 2]
```diff
@@ -838,11 +838,14 @@ private static String getCloudProviderPathFromDeploymentApplication(DeploymentAp
...
 
-	public TeamResource setManagerUserNames(TeamResource teamResource, String token) {
+	public TeamResource setManagerUsernamesAndEmails(TeamResource teamResource, String token) {
 		List<String> managerUserNames = new ArrayList<>();
+		List<String> managerEmails = new ArrayList<>();
 		try {
 			managerUserNames = this.domainService.getAllManagersFromDomain(teamResource.getDomainReference(), token)
 					.parallelStream().map(manager -> manager.getUserReference()).collect(Collectors.toList());
+			managerEmails = this.domainService.getAllManagersFromDomain(teamResource.getDomainReference(), token)
+					.parallelStream().map(manager -> manager.getEmail()).collect(Collectors.toList());
```
Comment (aniewielska) [inline src/main/java/uk/ac/ebi/tsc/portal/api/team/service/TeamService.java:848]: And as previously you are calling the same method from domainService twice!! Isn't the first call returning the value already?

Comment (navisatebi) [reply src/main/java/uk/ac/ebi/tsc/portal/api/team/service/TeamService.java]: Was an error oversight replaced it.

File: src/integration-test/java/uk/ac/ebi/tsi/portal/api/team/TeamRestControllerIT.java (hunk 1 above)
Comment (navisatebi) [reply src/integration-test/java/uk/ac/ebi/tsi/portal/api/team/TeamRestControllerIT.java]: Refactored name, non-team owners cannot see. The exception is not thrown here, removed it.

**Pull request merged**